src/main/java/org/dependencytrack/resources/v1/vo/BomSubmitRequest.java

```diff
@@ -110,6 +117,10 @@ public String getProjectVersion() {
         return projectVersion;
     }
 
+    public List<Tag> getProjectTags() {
```

Comment:
Can you add a `@Schema(description = "")` here, where `description` explains that the tags defined here will **overwrite** any tags on existing projects that were manually applied?

It was a discussion point in #1674 so I believe the API docs should call out the behavior.